plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -366,6 +366,50 @@ fu_plugin_uefi_capsule_update_splash (FuPlugin *plugin, FuDevice *device, GError
 	return fu_plugin_uefi_capsule_write_splash_data (plugin, device, image_bmp, error);
 }
 
+static gboolean
+fu_capsule_update_on_disk (FuDevice *device, GError **error)
+{
+	g_autofree guint64 *os_indications_supported = NULL;
+	g_autofree guint64 *os_indications = NULL;
```

Comment:
no need to autofree integers

Reply:
I am not sure I am following, plugins/uefi-capsule/fu-uefi-bootmgr.c does a similar thing while reading BootOrder.